index.html

```diff
@@ -55,97 +55,8 @@ <h2 class="secondtitle">I'm a software developer</h2>
-                        <button type="submit" class="button1">See Project</button>
-                    </div>
-                </div>
-            </div>
-            <br>
-        <div class="box cards1">
-                <div class="blank">
-                    <div class="cardscontainers">
-                        <p class="cardtitle">Multi-Post Stories<br>Gain+Glory</p>
-                        <ul class="ul1">
-                            <li class="ruby2">Ruby on rails</li>
-                            <li class="css0">css</li>
-                            <li class="java0">JavScript</li>
-                            <li class="html0">html</li>
-                        </ul>
-                        <button type="submit" class="button1">See Project</button>
-                    </div>
-                </div>
-            </div>
-            <br>
-            <div class="box cards1">
-                <div class="blank">
-                    <div class="cardscontainers">
-                        <p class="cardtitle">Multi-Post Stories<br>Gain+Glory</p>
-                        <ul class="ul1">
-                            <li class="ruby2">Ruby on rails</li>
-                            <li class="css0">css</li>
-                            <li class="java0">JavScript</li>
-                            <li class="html0">html</li>
-                        </ul>
-                        <button type="submit" class="button1">See Project</button>
-                    </div>
-                </div>
-            </div>
-            <br>
-            <div class="box cards1">
-                <div class="blank">
-                    <div class="cardscontainers">
-                        <p class="cardtitle">Multi-Post Stories<br>Gain+Glory</p>
-                        <ul class="ul1">
-                            <li class="ruby2">Ruby on rails</li>
-                            <li class="css0">css</li>
-                            <li class="java0">JavScript</li>
-                            <li class="html0">html</li>
-                        </ul>
-                        <button type="submit" class="button1">See Project</button>
-                    </div>
-                </div>
-            </div>
-            <br>
-            <div class="box cards1">
-                <div class="blank">
-                    <div class="cardscontainers">
-                        <p class="cardtitle">Multi-Post Stories<br>Gain+Glory</p>
-                        <ul class="ul1">
-                            <li class="ruby2">Ruby on rails</li>
-                            <li class="css0">css</li>
-                            <li class="java0">JavScript</li>
-                            <li class="html0">html</li>
-                        </ul>
-                        <button type="submit" class="button1">See Project</button>
-                    </div>
-                </div>
-            </div>
-        </div>
+        <div class="card"></div>
```

Comment:
- [ ] Your desktop version is fine but when I switch to the mobile version you don't have styling neither you have a pop-up for mobile so kindly implement it.
### your design
![his](https://user-images.githubusercontent.com/81727988/175564676-0f8ed7e5-ba90-4a5a-8598-d88799239512.png)

### Pop-Up design from Figma

![figma](https://user-images.githubusercontent.com/81727988/175564847-5a5a53ca-4635-4e75-aa40-d4851eff2767.png)